### PR TITLE
Add capture_error method to Python base SDK

### PR DESCRIPTION
**Pull request by selcukcihan**
Related issue https://linear.app/serverless/issue/SC-561/report-detailed-error-information-in-case-of-error-resolution

### Description
* Add `serverlessSdk.capture_error` method
* Add documentation for the new method

### Testing done
Unit tested

### CODE REVIEW

**Comment by selcukcihan** [comment]
@medikoo ready for review please 🙏

File: python/packages/sdk/tests/test_sdk.py [hunk 1]
```diff
@@ -90,3 +90,15 @@ def test_create_trace_span_returns_trace_span(sdk: ServerlessSdk):
...
+def test_sdk_exposes_capture_error(sdk: ServerlessSdk):
+    # given
+    error = Exception("My error")
+
+    # when
+    captured = sdk.capture_error(error, tags={"user.tag": "somevalue"})
+
+    # then
+    assert captured.tags["error.message"] == "My error"
+    assert captured.custom_tags["user.tag"] == "somevalue"
```
Comment (medikoo) [inline python/packages/sdk/tests/test_sdk.py:111]: Can we also confirm on type here

**Review by medikoo — CHANGES_REQUESTED** [review]
Thank you @selcukcihan 👍

**Review by medikoo — APPROVED** [review]
Looks very good 👍 

Let's follow up with the release, and update to the AWS Lambda SDK and release over there.

Great work @selcukcihan 💯

**Comment by selcukcihan** [comment]
> Looks very good 👍
> 
> Let's follow up with the release, and update to the AWS Lambda SDK and release over there.
> 
> Great work @selcukcihan 💯

Thanks @medikoo Release PR => https://github.com/serverless/console/pull/522

**Pull request merged**